### PR TITLE
updating olm config to link to openshift preinstallation steps

### PR DESCRIPTION
**Pull request by acornett21**
Issue #, if available:
N/A
Description of changes:
Updating the `olmconfig.yaml` to have a link to the preinstallation steps for OpenShift, as well as adding in the maintainers name/email. This information is used when generating the bundle.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Signed-off-by: Adam D. Cornett <adc@redhat.com>

### CODE REVIEW

File: olm/olmconfig.yaml [hunk 1]
```diff
@@ -18,13 +18,19 @@ description: |-
 
 
   This controller is a component of the [AWS Controller for Kubernetes](https://github.com/aws/aws-controllers-k8s)
-  project. This project is currently in **developer preview**. 
+  project. This project is currently in **developer preview**.
+
+
+  **Pre-Instaliation Steps**
```
Comment (jaypipes) [inline olm/olmconfig.yaml:24]: s/Instaliation/Installation/ :)

**Review by jaypipes — CHANGES_REQUESTED** [review]
little typo inline... though I do like the sound of "Instaliation" :) Sounds much more official than "installation".

File: olm/olmconfig.yaml (hunk 1 above)
Comment (acornett21) [reply olm/olmconfig.yaml]: Nice catch! Looks like I don't get spellcheck in `yaml` thought I did. :(

**Review by jaypipes — APPROVED** [review]
:+1:

**Pull request merged**